core-httpclient-impl/src/main/java/com/optimizely/ab/odp/DefaultODPApiManager.java

```diff
@@ -0,0 +1,179 @@
+        StringBuilder segmentsString = new StringBuilder();
+        for (int i = 0; i < segmentsList.size(); i++) {
+            if (i > 0) {
+                segmentsString.append(", ");
+            }
+            segmentsString.append("\\\"").append(segmentsList.get(i)).append("\\\"");
+        }
+        return segmentsString.toString();
+    }
+
+    // ODP GraphQL API
+    // - https://api.zaius.com/v3/graphql
+    // - test ODP public API key = "W4WzcEs-ABgXorzY7h1LCQ"
+    /*
+
+     [GraphQL Request]
+
+     // fetch info with fs_user_id for ["has_email", "has_email_opted_in", "push_on_sale"] segments
+     curl -i -H 'Content-Type: application/json' -H 'x-api-key: W4WzcEs-ABgXorzY7h1LCQ' -X POST -d '{"query":"query {customer(fs_user_id: \"tester-101\") {audiences(subset:[\"has_email\",\"has_email_opted_in\",\"push_on_sale\"]) {edges {node {name state}}}}}"}' https://api.zaius.com/v3/graphql
+     // fetch info with vuid for ["has_email", "has_email_opted_in", "push_on_sale"] segments
+     curl -i -H 'Content-Type: application/json' -H 'x-api-key: W4WzcEs-ABgXorzY7h1LCQ' -X POST -d '{"query":"query {customer(vuid: \"d66a9d81923d4d2f99d8f64338976322\") {audiences(subset:[\"has_email\",\"has_email_opted_in\",\"push_on_sale\"]) {edges {node {name state}}}}}"}' https://api.zaius.com/v3/graphql
+     query MyQuery {
+       customer(vuid: "d66a9d81923d4d2f99d8f64338976322") {
+         audiences(subset:["has_email","has_email_opted_in","push_on_sale"]) {
+           edges {
+             node {
+               name
+               state
+             }
+           }
+         }
+       }
+     }
+     [GraphQL Response]
+
+     {
+       "data": {
+         "customer": {
+           "audiences": {
+             "edges": [
+               {
+                 "node": {
+                   "name": "has_email",
+                   "state": "qualified",
+                 }
+               },
+               {
+                 "node": {
+                   "name": "has_email_opted_in",
+                   "state": "qualified",
+                 }
+               },
+                ...
+             ]
+           }
+         }
+       }
+     }
+
+     [GraphQL Error Response]
+     {
+       "errors": [
+         {
+           "message": "Exception while fetching data (/customer) : java.lang.RuntimeException: could not resolve _fs_user_id = asdsdaddddd",
+           "locations": [
+             {
+               "line": 2,
+               "column": 3
+             }
+           ],
+           "path": [
+             "customer"
+           ],
+           "extensions": {
+             "classification": "InvalidIdentifierException"
+           }
+         }
+       ],
+       "data": {
+         "customer": null
+       }
+     }
+    */
+    @Override
+    public String fetchQualifiedSegments(String apiKey, String apiEndpoint, String userKey, String userValue, List<String> segmentsToCheck) {
```

Comment:
Name "fetchQualifiedSegments" is already used in upcoming "ODPSegmentManager" (per Jae's naming). Risk of name conflict. Unless you use different names, but that might be a deviation from naming consistency across SDKs?